commands.py

```diff
@@ -36,6 +37,7 @@ def __init__(self, maestrocommand, commandvalue):
 MAESTRO_COMMANDS.append(MaestroCommand('Power', 34, 'onoff40'))
 MAESTRO_COMMANDS.append(MaestroCommand('Fan_State', 37, 'int'))# 0, 1, 2, 3 ,4,  5 ,6
 MAESTRO_COMMANDS.append(MaestroCommand('Control_Mode', 40, 'onoff')) # 0 = Auto , 1 = Manual
+MAESTRO_COMMANDS.append(MaestroCommand('Set_DateTime', 0, 'datetime')) # Format ddmmYYYYHHMM
```

Comment:
Not absolutely shure if the id should be 0 or 1. Below is the original message send to the stove from the app:
`42["chiedo",{"serialNumber":"1901290060006","macAddress":"86F3EB6DEF87","tipoChiamata":1,"richiesta":"C|SalvaDataOra|161220201128"}]`

Reply:
MAESTRO_COMMANDS  are meant to privide a list of WriteParametri command ID's. So, power on of will be C|WriteParametri|34|

The set datetime is a diiferent command by itself. It is 
C|SalvaDataOra|

0 or 1 will have no meaning there